### ISSUE TITLE
"gcp-billing-data-users@" Group is needed. Prerequisites in documentation missing

### ISSUE THREAD

**Issue by julianwieg** [terraform-google-modules/terraform-example-foundation#918]
### TL;DR

In the 1-org  readme.md  there is no mention that the gcp-billing-data-users@<example.com> group must exist in Cloud Identity or Google Workspace for the TF Apply to work.

### Expected behavior

Better documentation (for noob users like me)

### Observed behavior

Error: Request `Create IAM Members roles/bigquery.user group:gcp-billing-data-users@domain.com for project "prj-c-billing-logs-762222"` returned error: Error applying IAM policy for project "prj-c-billing-logs-76222": Error setting IAM policy for project "prj-c-billing-logs-76222": googleapi: Error 400: Group gcp-billing-data-users@domain.com does not exist., badRequest
│ 
│   with google_project_iam_member.billing_bq_user,
│   on iam.tf line 83, in resource "google_project_iam_member" "billing_bq_user":
│   83: resource "google_project_iam_member" "billing_bq_user" {

### Terraform Configuration

```hcl
see above. config is fine but I only noticed this group does not exist after TF apply
```

### Terraform Version

```sh
see above
```
